server/svix-server/src/cfg.rs

```diff
@@ -243,6 +247,30 @@ impl ConfigurationInner {
             CacheType::RedisCluster => CacheBackend::RedisCluster(self.cache_dsn().expect(err)),
         }
     }
+
+    pub fn shared_store_backend(&self) -> SharedStoreBackend<'_> {
+        let err = "Called [`shared_store_backend`] before validating configuration";
+
+        match self.shared_store_type() {
+            SharedStoreType::None => SharedStoreBackend::None,
```

Comment:
None is not allowed for the shared store. So I guess set it to at least memory if the cache is set to None.